search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/AuthorizationFilterTransformer.java

```diff
@@ -19,14 +19,14 @@ public class AuthorizationFilterTransformer implements FilterTransformer<Authori
   @Override
   public SearchQuery toSearchQuery(final AuthorizationFilter filter) {
     return and(
-        filter.ownerKey() == null ? null : term("value.ownerKey", filter.ownerKey()),
-        filter.ownerType() == null ? null : term("value.ownerType", filter.ownerType()),
-        filter.resourceKey() == null ? null : term("value.resourceKey", filter.resourceKey()),
-        filter.resourceType() == null ? null : term("value.resourceType", filter.resourceType()));
+        filter.ownerKey() == null ? null : term("ownerKey", filter.ownerKey()),
```

Comment:
@Ben-Sheppard can you please update [AuthorizationEntity](https://github.com/camunda/camunda/blob/main/search/search-domain/src/main/java/io/camunda/search/entities/AuthorizationEntity.java) at the same time and remove the `value` intermediate field ?

Reply:
I can indeed, but there are some other changes that I think will be needed with changing that class so it would be best if I follow up with a PR for those (to keep things smaller, this PR has already bloated a bit!)